# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/uclibc/hotfix-cris.patch.cross0
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- uClibc-ng-1.0.42/Rules.mak.vanilla	2023-01-03 17:57:06.326407449 +0100
+++ uClibc-ng-1.0.42/Rules.mak	2023-01-03 17:57:15.625406837 +0100
@@ -442,7 +442,7 @@
 ifeq ($(TARGET_ARCH),cris)
 	CPU_LDFLAGS-$(CONFIG_CRIS)+=-Wl,-mcrislinux
 	CPU_LDFLAGS-$(CONFIG_CRISV32)+=-Wl,-mcrislinux
-	CPU_CFLAGS-$(CONFIG_CRIS)+=-mlinux
+	#CPU_CFLAGS-$(CONFIG_CRIS)+=-mlinux
 	PICFLAG:=-fpic
 	PIEFLAG_NAME:=-fpie
 endif
